tools/getkeys/index.js

```diff
@@ -1,18 +1,17 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-constant-condition */
+/* eslint-disable no-return-await */
```

Comment:
Please prefer line-level disables to full-file.

Reply:
Got it! Removed them.